lib/srv/regular/sshserver.go

```diff
@@ -349,6 +352,23 @@ func (s *Server) GetSELinuxEnabled() bool {
 	return s.enableSELinux
 }
 
+// ChildLogConfig returns the child log config.
+func (s *Server) ChildLogConfig() srv.ChildLogConfig {
+	if s.childLogConfig != nil {
+		return *s.childLogConfig
+	}
+
+	// Default to stderr debug logs (tests)
```

Comment:
Is it really safe to assume that if we have no `childLogConfig` that we are in a test context? Can we instead update tests to set a child logging config if we want test logs?